src/js/step.js

```diff
@@ -413,7 +372,7 @@ export class Step extends Evented {
    *
    * @private
    */
-  _setUpButtons() {
+  _setupButtonOpts() {
```

Comment:
I've let some renaming slide, but this one seems arbitrary to me. Isn't a function that sets up buttons being named `setUpButtons` okay? 😛

Reply:
Totally. I don't remember why I did that, tbh. Perhaps I thought is was returning opts instead of setting `this.options.buttons` directly 🙃.

In any case, I changed it back.